tests/test_wallet.py

```diff
@@ -376,6 +378,25 @@ async def test_full_exit(self):
             decimals=18)
         assert await self.ethereum_provider.full_exit(token, 6713)
 
+    async def test_full_exit_nft(self):
+        """
+        INFO: made by getting all NFT by corresponded address & dumping,
+                Symbol: 'NFT-70848'
+                '70848'
+                address: '0x5e71f0f9b891f22d79ff8697dd4e3e0db371cda5'
+                creator_address: '0x995a8b7f96cb837533b79775b6209696d51f435c'
+                id: 70848
+                account_id: 36357
+        """
+        account_id = 36357
+        token = Token(
+            address=Web3.toChecksumAddress('0x5e71f0f9b891f22d79ff8697dd4e3e0db371cda5'),
+            id=70848,
+            symbol='NFT-70848',
+            decimals=0
+        )
+        assert await self.ethereum_provider.full_exit_nft(token, account_id)
```

Comment:
This shouldn't actually work more than one time. Once an NFT is fullexit-ed to l1, you can not fullexit it once again :)

The test itself will work, though, because you are allowed to call full exit nft multiple times, even though there is nothing to full-ext. This is ok, but I just wanted you to know

Reply:
Ok. Thank you for explanation.